doc/classes/CanvasTexture.xml

```diff
@@ -5,7 +5,7 @@
 	</brief_description>
 	<description>
 		[CanvasTexture] is an alternative to [ImageTexture] for 2D rendering. It allows using normal maps and specular maps in any node that inherits from [CanvasItem]. [CanvasTexture] also allows overriding the texture's filter and repeat mode independently of the node's properties (or the project settings).
-		[b]Note:[/b] [CanvasTexture] cannot be used in 3D rendering. For physically-based materials in 3D, use [BaseMaterial3D] instead.
+		[b]Note:[/b] [CanvasTexture] should not be used in 3D. It will not display correctly when applied to any [VisualInstance3D], such as [Sprite3D] or [Decal]. For physically-based materials in 3D, use [BaseMaterial3D] instead.
```

Comment:
```suggestion
		[b]Note:[/b] [CanvasTexture] cannot be used in 3D. It will not display correctly when applied to any [VisualInstance3D], such as [Sprite3D] or [Decal]. For physically-based materials in 3D, use [BaseMaterial3D] instead.
```

Looks good. I would go back to "cannot" as it is stronger. Should not implies that it might work but you shouldn't do it. In this case it just doesn't work, its the same as not applying a texture